doc/nrf/releases_and_maturity/releases/release-notes-1.8.0.rst

```diff
@@ -195,7 +195,7 @@ nRF Desktop
 
   * Added documentation for :ref:`nrf_desktop_usb_state_pm`.
   * Added :ref:`nrf_desktop_ble_state_pm`.
-  * Added peer erase feature for the ``nrf52840dk_nrf52840`` build target.
+  * Added peer erase feature for the ``nrf52840dk_nrf52840`` board target.
```

Comment:
I think we should not change terminology in release notes

Reply:
Agreed.